Improve initial section and supported devices for Plugwise (#36500)
home-assistant/home-assistant.github.io: source/_integrations/plugwise.markdown

ha_integration_type: hub
---

> [Plugwise](https://www.plugwise.com) provides smart home climate and power monitoring devices. This integration allows you to monitor and control your climate, energy consumption (including gas) consumption, and energy production. The energy information can be used for the [energy dashboard](/home-energy-management).

Reviewer: ... provides smart home devices that allow you to monitor and control your climate, energy (including gas) consumption, and energy production.

> This integration supports one or multiple Plugwise Gateways available on your network. You can connect to these gateways using your browser, the Plugwise App, or Home Assistant. There are 4 [supported](#supported-devices) types of gateways:

Reviewer: ... **connected** to your network.

> - Full zonecontrol using the [Adam](https://www.plugwise.com/en_US/zonecontrol) using [additional devices](#adam) such as smart valves and smart-plugs.

Reviewer: ... such as zone thermostats, smart valves ...

{% include integrations/config_flow.md %}

{% configuration_basic %}
Host:
  description: "The hostname or IP address of your gateway. For example: `192.168.1.25`. You can find it in your router or in the Plugwise app using the **Settings** icon (&#9776;) -> **System** -> **Network**. In the Plugwise App, to locate a specific device, select **Gateways** on the main screen, choose your desired gateway, and then follow the previous instructions. Normally, the gateway(s) are automatically discovered, and you don't have to provide the hostname or IP address."
Username:
  description: "Username to log in to the gateway. This should be 'smile' for most devices, or 'stretch' for a Stretch."
Password:
  description: "Each gateway requires its unique 8-character ID, found on a sticker at the bottom, as its password."
{% endconfiguration_basic %}

### Schedule Management

1. **Initial Setup**: First, activate a schedule using the Plugwise App or browser.
2. **Control via Home Assistant**:
   - Use the climate card to activate/deactivate schedules.
   - 'Auto' mode indicates the schedule is active.

>    - 'Heat' mode signifies the schedule is inactive.

Reviewer: Heat, Cool or Heat_cool mode ...

Author: Is this better?

  - An under-floor heating controller Koen (note: a Koen always comes with a Plug, which is the active part),
  - Smart switches, either Plug or Aqara Smart Plug.

### Anna

The current implementation of the Python module (Plugwise-Smile) includes:
A smart thermostat, supporting:

Adam (zone_control) with On/Off, OpenTherm, and Loria/Thermastage heating and cooling support:
- OnOff, OpenTherm heating and Elga or Loria/Thermastage with heating and cooling support.
- Running firmware v4.x, v3.x or v1.x.

 - v3.x
 - v2.3
### P1 (DSMR)

 - Devices supported are Anna, Lisa, Jip, Floor, Tom, Plug, Aqara Smart Plug, and Koen (a Koen always comes with a Plug, the active part)
A smart meter monitor for single or multiphase P1 and gas usage monitoring with the P1 running firmware v4.x, v3.x or v2.x.

Anna (thermostat) with OnOff, OpenTherm heating, and Elga and Loria/Thermastage with heating and cooling support:
### Stretch (end-of-sale)

 - v4.x
 - v3.x
 - v1.x
For legacy power switches, such as the Circles or Stealths, with v3.x or v2.x Stretch firmware.

On the Elga, the cooling-mode can only be turned on, or off, via a switch present on the device, not via a toggle in the Plugwise App.

> Please make sure to reload the Plugwise integration after the cooling-mode-switch is turned off after being on, or the other way around. This will ensure that the Plugwise integration is being adapted to the change in function of the Elga.

Author: Removing this for now, it will return as part of the 'known limitations' section already suggested and ready for upcoming PR (to keep this one small) - meanwhile as discussed it's there in #36504

> ### Unsupported devices

Author: @frenck or should this be a level 2 indent?

          preset_mode: asleep
```

## Supported devices

### Adam

> A complete zone control system also known as Adam HA, supporting:

Reviewer: Add info about the Adam SA (Stand Alone, no ZigBee stick present)  to make a single, basic room thermostat smart?

Author: Please crosscheck

Author: Is it ok now?  (If so, please mark resolved)

- On/Off, OpenTherm or Loria/Thermastage heating, and cooling support.
- Running firmwares v3.x or v2.3.
- Additional devices:
  - Zone thermostats such as Lisa or Anna (see warning below on Anna),
  - A temperature sensor, Jip,
  - Valve controllers called Floor or Tom,
  - An under-floor heating controller Koen (note: a Koen always comes with a Plug, which is the active part),
  - Smart switches, either Plug or Aqara Smart Plug.

### Anna

A smart thermostat, supporting:

- On/Off, OpenTherm heating, and Elga or Loria/Thermastage with heating and cooling support.
- Running firmware v4.x, v3.x or v1.x.

### P1 (DSMR)

> A smart meter monitor for single or multiphase P1 and gas usage monitoring with the P1 running firmware v4.x, v3.x or v2.x.

Reviewer: A P1 smart meter monitor for the single- or multi-phase grid power connection to your home ...

### Stretch (end-of-sale)

> For legacy power switches, such as the Circles or Stealths, with v3.x or v2.x Stretch firmware.

Reviewer: For controlling and monitoring legacy ...

          preset_mode: asleep
```

## Supported devices

> The Plugwise integration relies on the [plugwise](https://pypi.org/project/plugwise/) module for Python. It currently provides support for:

Reviewer 2: There is no reason the link end-user into libraries here. It doesn't add anything and merely may cause a technical scare. Let's remove it.

### Adam

A complete zone control system also known as Adam HA, supporting:

> - On/Off, OpenTherm or Loria/Thermastage heating and cooling support.

Reviewer 2: ```suggestion
- On/Off, OpenTherm or Loria/Thermastage heating, and cooling support.
```

> - Running firmwares v3.x or v2.3

Reviewer 2: ```suggestion
- Running firmwares v3.x or v2.3.
```

- Additional devices:
  - Zone thermostats such as Lisa or Anna (see warning below on Anna),
  - A temperature sensor, Jip,
  - Valve controllers called Floor or Tom,
  - An under-floor heating controller Koen (note: a Koen always comes with a Plug, which is the active part),
  - Smart switches, either Plug or Aqara Smart Plug.

### Anna

A smart thermostat, supporting:

> - OnOff, OpenTherm heating and Elga or Loria/Thermastage with heating and cooling support.

Reviewer 2: ```suggestion
- On/Off, OpenTherm heating, and Elga or Loria/Thermastage with heating and cooling support.
```

- Running firmware v4.x, v3.x or v1.x.

### P1 (DSMR)

A smart meter monitor for single or multiphase P1 and gas usage monitoring with the P1 running firmware v4.x, v3.x or v2.x.

### Stretch (end-of-sale)

For legacy power switches, such as the Circles or Stealths, with v3.x or v2.x Stretch firmware.

### Unsupported devices

#### Anna P1

> A smart thermostat, providing abundant solar energy billing by transferring abundant engergy in heatpump usage. If you have one, please let us know by opening a [feature request](https://github.com/plugwise/python-plugwise/issues/new/choose).

Reviewer 2: ```suggestion
A smart thermostat provides abundant solar energy billing by transferring abundant energy in heat pump usage. If you have one, please let us know by opening a [feature request](https://github.com/plugwise/python-plugwise/issues/new/choose).
```

Reviewer 2: We shouldn't ask users to make a feature request outside of the Home Assistant project. Let's remove that.

Author: I've rewritten it - mostly because the providing/provides didn't work out (although I could have misread their website and it is a thermostat generating solar) :)

#### Stick

> Plugwise formerly sold power-based products comprised of a USB stick, as the controller, and smart plugs (among a few other items). This integration does not support the USB-stick. Reuse of the these products, such as Circles and Stealths using a Stretch or an Adam is supported. Work for USB support is in development by the community, but not ready to become a formal Home Assistant integration just yet.

Reviewer 2: Some wording and the removal of expectations set in the last sentence.

```suggestion
Plugwise formerly sold power-based products comprised of a USB stick as the controller and smart plugs (among a few other items). This integration does not support the USB stick. Reuse of these products, such as Circles and Stealths using a Stretch or an Adam, is supported.
```

          preset_mode: asleep
```

## Supported devices

### Adam

A complete zone control system also known as Adam HA, supporting:

> - On/Off, OpenTherm or Loria/Thermastage heating, and cooling support.

Reviewer: Maybe take a bit more info from here: https://www.plugwise.com/product/adam-ha/?lang=en

Author: Reworked new comments by just adding the links to the website (forgoing the need to be fully copy/pasting). As a 'supported devices' section I'd assume it's a mere list of things supported. If we are missing things it should go in the top description I'd suppose (for that is what the user will mostly read if interested, supported devices is more when you want to make sure things you buy/bought are going to be supported?)

Author: Could you check if the intro and links on top would be sufficient? (If so, please mark resolved)

- Running firmwares v3.x or v2.3.
- Additional devices:
  - Zone thermostats such as Lisa or Anna (see warning below on Anna),
  - A temperature sensor, Jip,
  - Valve controllers called Floor or Tom,
  - An under-floor heating controller Koen (note: a Koen always comes with a Plug, which is the active part),
  - Smart switches, either Plug or Aqara Smart Plug.

{% note %}

> You can also use the Adam SA (Stand-alone, no Zigbee stick present) to make a single zone thermostat. 

Reviewer: .. to make your regular OpenTherm thermostat smart.

This is how it's described on the Plugwise webpage

{% endnote %}

{% endtip %}

> ### Anna

Reviewer: Can we provide a link here? https://www.plugwise.com/product/anna/?lang=en
Because below one-liner is far from complete.
